### PR TITLE
Update GitHub templates

### PR DESCRIPTION
**Pull request by jemrobinson**
## :white_check_mark: Checklist

- [x] You have given your pull request a meaningful title (_e.g._ `Enable foobar integration` rather than `515 foobar`).
- [x] You are targeting the appropriate branch. If you're not certain which one this is, it should be **`develop`**.
- [x] Your branch is up-to-date with the **target branch** (it probably was when you started, but it may have changed since then).

### :vertical_traffic_light: Depends on

n/a

### :arrow_heading_up: Summary

Update GitHub templates for PRs and issues

### :closed_umbrella: Related issues

n/a

### :microscope: Tests

Documentation only

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/deployment_bug_report.md [hunk 1]
```diff
@@ -20,12 +20,24 @@ Before reporting a problem please check the following. Replace the empty checkbo
...
-## :cactus: Powershell module versions
+## :cactus: Package dependencies
 
-<!-- Paste the output of deployment/CheckRequirements.ps1 here -->
+<details>
+<summary>List of packages</summary>
+
+<!--
+Powershell: Run `deployment/CheckRequirements.ps1` and paste the output below
+Python: Run `pip freeze` and paste the output below
```
Comment (JimMadge) [inline .github/ISSUE_TEMPLATE/deployment_bug_report.md]:
```suggestion
Python: Run `hatch run pip freeze` and paste the output below
```

If people are following our instructions that might be better advice.

File: .github/ISSUE_TEMPLATE/deployment_bug_report.md [hunk 2]
```diff
@@ -20,12 +20,24 @@ Before reporting a problem please check the following. Replace the empty checkbo
 
 ## :computer: System information
 
-- **Operating System:** <!-- Which operating system are you using: Windows, Linux, macOS etc. -->
-- **Data Safe Haven version:** <!-- Which version of the Data Safe Haven are you using: 3.0.0, 3.1.1 etc. -->
+- **Operating System:** <!-- Which operating system you are using: Windows, Linux, macOS etc. -->
+- **Data Safe Haven version:** <!-- Which version of the Data Safe Haven you are using: 3.0.0, 3.1.1 etc. If this is on a branch use `branch name @ 7-letter commit-id` -->
 
-## :cactus: Powershell module versions
+## :cactus: Package dependencies
```
Comment (JimMadge) [inline .github/ISSUE_TEMPLATE/deployment_bug_report.md]:
```suggestion
## :cactus: Packages
```

**Review by JimMadge — CHANGES_REQUESTED** [review]
Just a couple of suggestions for text changes.

If we'd like to use those, they will need to be applied in the other templates too.

File: .github/ISSUE_TEMPLATE/deployment_bug_report.md (hunk 1 above)
Comment (jemrobinson) [reply .github/ISSUE_TEMPLATE/deployment_bug_report.md]: Done in 5133ea6ee

File: .github/ISSUE_TEMPLATE/deployment_bug_report.md (hunk 2 above)
Comment (jemrobinson) [reply .github/ISSUE_TEMPLATE/deployment_bug_report.md]: Done in 3601ef50d

**Review by JimMadge — APPROVED** [review]

**Pull request merged**